test/unit/HeartbeatTest.js

```diff
@@ -0,0 +1,36 @@
+// Third-party dependencies
+const { expect, use } = require('chai')
+const { describe, it } = require('mocha')
+const sinonChai = require('sinon-chai')
+
+// In-house dependencies
+const index = require('../../index')
+
+// Configure Chai
+use(sinonChai)
+
+describe('HeartbeatTest.js unit tests', () => {
```

Comment:
And here, this should be the name of the file and the function name, like: `index.js unit tests: convertStateArrayToObject`, similar to what is done in `test/unit/BraveAlerterConfiguratorTest/alertSessionChangedCallbackTest.js`. This will make it easy for someone to find what code is having problems if any of these tests fail.